lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runtime/azbatch/util/command/WindowsCommandBuilder.java

```diff
@@ -71,4 +74,17 @@ protected String getEvaluatorShimClasspath() {
     return String.format("'%s'", StringUtils.join(
         super.classpathProvider.getEvaluatorClasspath(), CLASSPATH_SEPARATOR_CHAR));
   }
+
+  @Override
+  public String getIpAddressFilePath() {
+    return "%AZ_BATCH_JOB_PREP_WORKING_DIR%\\hostip.txt";
+  }
+
+  @Override
+  public String captureIpAddressCommandLine() {
+    LOG.log(Level.INFO, "Inside java code for WindowsCommandBuilder");
+    return String.format("powershell /c \"Set-Content -Path %s -Value "
```

Comment:
Log needs to be removed or improved..

Reply:
Removed.

---
In reply to: [200421622](https://github.com/sharathmalladi/reef/pull/6#discussion_r200421622) [](ancestors = 200421622)